baklava/package.json

```diff
@@ -1,57 +1,70 @@
 {
-  "name": "dogehouse",
-  "version": "1.0.14",
-  "description": "Taking voice conversations to the moon 🚀",
-  "main": "./dist/electron.js",
-  "scripts": {
-    "compile": "tsc",
-    "watch": "tsc -w",
-    "dev": "npm run compile && node ./dist/dev.js",
-    "start": "npm run compile && electron ./dist/electron.js",
-    "build": "npm run compile && electron-builder",
-    "build:all": "npm run compile && electron-builder -mwl --win msi",
-    "build:mac": "npm run compile && electron-builder --mac",
-    "build:win": "npm run compile && electron-builder --win msi",
-    "build:linux": "npm run compile && electron-builder --linux"
-  },
-  "keywords": [
-    "DogeHouse",
-    "Electron"
-  ],
-  "author": "Ben Awad <benawadapps@gmail.com>",
-  "homepage": "https://github.com/benawad/dogehouse/",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/benawad/dogehouse"
-  },
-  "bugs": {
-    "url": "https://github.com/benawad/dogehouse/issues"
-  },
-  "license": "MIT",
-  "devDependencies": {
-    "@types/node": "^14.14.31",
-    "electron": "^11.3.0",
-    "electron-builder": "^22.10.5",
-    "typescript": "^4.2.2"
-  },
-  "build": {
-    "appId": "com.electron.dogehouse",
-    "productName": "DogeHouse",
-    "asar": true,
-    "mac": {
-      "icon": "icons/icon.png",
-      "category": "public.app-category.social-networking",
-      "entitlements": "settings/entitlements.mac.plist",
-      "extendInfo": {
-        "NSMicrophoneUsageDescription": "Mic access for speeach input in voice."
-      },
-      "target": [
-        "dmg"
-      ]
+    "name": "dogehouse",
+    "version": "1.0.0",
```

Comment:
The version number should not go backwards. Again, I think you have an outdated fork